python/pyhail/dataset.py

```diff
@@ -1776,10 +1963,20 @@ def repartition(self, npartition, shuffle=True):
         """Increase or decrease the dataset sharding.  Can improve performance
         after large filters.
 
+        **Examples**
+
+        Decrease the number of partitions by 50%
```

Comment:
Can we have a repartition to fixed size?  I think that's the most common use case.